.github/workflows/test.yml

```diff
@@ -27,7 +30,9 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get -qq update
-          sudo apt-get install -y squashfs-tools
+          sudo apt-get install -y squashfs-tools libcap-dev libacl1-dev
+          sudo add-apt-repository -y ppa:dqlite/dev
```

Comment:
Why does distrobuilder need dqlite?

Reply:
It's probably importing something from `github.com/lxc/lxd` which is triggering this, but it'd definitely be preferable to track down what's causing it and split packages differently on the LXD side instead of pulling all of this into distrobuilder.

Reply:
Probably the apparmor package, yeah would be good to split.

Reply:
The new `Unpack()` function takes a `sys.OS` argument. The `sys` package requires the `db` package among others. So that's the reason. `distrobuilder` itself doesn't need either of the mentioned packages.

Reply:
Perhaps allowing `nil` for that specific argument would solve this.

Reply:
That explains dqlite, but not liblxc?

Can we decouple `sys` from `db` or move `sys.OS` elsewhere?

Reply:
Depends how the liblxc dep is being introduced.